lib/cequel/record/properties.rb

```diff
@@ -262,7 +262,7 @@ def attribute_names
       def attributes
         attribute_names.each_with_object({}) do |name, attributes|
           attributes[name] = read_attribute(name)
-        end
+        end.with_indifferent_access
```

Comment:
I think it would be a bit more efficient to just initialize a `HashWithIndifferentAccess` on line 263, rather than building up a normal `Hash` as an intermediate structure and then converting it to a `HashWithIndifferentAccess` here.